src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Amazon API Gateway V2 Monitoring Integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS API Gateway v2 monitoring integration: what data it reports, and how to enable it."
+freshnessValidatedDate: 2023-07-10
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) now include an integration for AWS API Gateway V2, sending its metrics data to New Relic.
+
+This document explains the integration's features, how to activate it, and what data you can report.
+
+## Features [#features]
+
+Collect and send telemetry data to New Relic from your [Amazon API Gateway v2](https://aws.amazon.com/api-gateway/) using our integration. Check your services, query incoming data, and build dashboards to observe everything at a glance.
+
+## Activate integration [#activate-integration]
+
+To enable this integration, see how to [connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).
+
+## Find and use data [#find-use-data]
+
+To find your integration's metrics, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Metrics & Events** and filter by `aws.apigateway`.
+
+## Metric data [#metric-data]
+
+This New Relic infrastructure integration collects the following Amazon API Gateway v2 metrics:
+
+### Amazon API Gateway v2 Metric data
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+      Metric  
+      </th>
+      <th>
+      Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+      `4xx`
+      </td>
+      <td>
+      The number of client-side errors captured in a given period.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `5xx`
+      </td>
+      <td>
+      The number of server-side errors captured in a given period.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Count`
+      </td>
+      <td>
+      The total number API requests in a given period.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `IntegrationLatency`
+      </td>
+      <td>
+      The time between when API Gateway relays a request to the backend and when it receives a response from the backend.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Latency`
+      </td>
+      <td>
+      The time between when API Gateway receives a request from a client and when it returns a response to the client. The latency includes the integration latency and other API Gateway overhead.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `DataProcessed`
+      </td>
+      <td>
+      The amount of data processed in bytes.
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+## Dimensions
```

Comment:
```suggestion
## Dimensions [#dimensions]
```

Reply:
@rhetoric101, I made the suggested change.